src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -417,6 +417,12 @@ public static void MapContainerView(IViewHandler handler, IView view)
 				handler.HasContainer = viewHandler.NeedsContainer;
 			else
 				handler.HasContainer = view.NeedsContainer();
+
+			handler.UpdateValue(nameof(IView.Visibility));
+   
+   			#if WINDOWS
+			handler.UpdateValue(nameof(IView.Opacity));
```

Comment:
Do we know if it is just windows only? Somehow I feel opacity is such a basic feature it affects more and we just saw it with windows first.

If we don't want to move the cheese to much in the PR, maybe just make this comment an issue and I can have a look when I get back to my keyboard.

Reply:
https://github.com/dotnet/maui/issues/23418